providers/defltprov.c

```diff
@@ -396,6 +396,26 @@ static const OSSL_ALGORITHM deflt_signature[] = {
     { PROV_NAMES_DSA, "provider=default", ossl_dsa_signature_functions },
 #endif
     { PROV_NAMES_RSA, "provider=default", ossl_rsa_signature_functions },
+#if !defined(OPENSSL_NO_MD5) && !defined(FIPS_MODULE)
+    { PROV_NAMES_RSA_MD5, "provider=default", ossl_rsa_md5_signature_functions },
+#endif
+#if !defined(OPENSSL_NO_RMD160) && !defined(FIPS_MODULE)
```

Comment:
Hmm... actually the fips provider has its own table - so the fips approved algorithms should be added to the fips/fipsprov.c as well.